### PR TITLE
fix device issue during calibration

### PR DESCRIPTION
**Pull request by xin3he**

### CODE REVIEW

**Comment by xin3he** [comment]
https://inteltf-jenk.sh.intel.com/job/inc_gaudi_test/154/

**Comment by xin3he** [comment]
https://inteltf-jenk.sh.intel.com/job/inc_gaudi_test/155/

**Comment by xin3he** [comment]
This PR fixs error during calibration: [gaudi2][1.19][GPTQ] RuntimeError: Expected all tensors to be on the HPU device, but found at least one input[idx=2] on cpu (details: torch.LongTensor)

File: examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py [hunk 1]
```diff
@@ -403,11 +403,12 @@ def calib_func(prepared_model):
             max_seq_length=args.gptq_max_seq_length,
         )
         dataloader_for_calibration = dataloaderPreprocessor.get_prepared_dataloader()
-        from neural_compressor.torch.algorithms.weight_only.utility import move_input_to_device
+        from neural_compressor.torch.utils import get_model_device, move_input_device
```
Comment (thuang6) [inline examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py:406]: I saw this move_input_to_device function implemented/copied in multiple location e.g.. weight-only/smoothquant.  it's better to be replaced with this single mov_input_device function under general utily (mayber after this release)

Comment (xin3he) [reply examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py:406]: @Kaihui-intel FYI. I had moved these functions to general utils.

**Pull request merged**